numpyro/contrib/nested_sampling.py

```diff
@@ -298,8 +307,7 @@ def get_weighted_samples(self):
                 "NestedSampler.run(...) method should be called first to obtain results."
             )
 
-        num_samples = self._results.total_num_samples
-        return self._results.samples, self._results.log_dp_mean[:num_samples]
+        return self._results.samples, self._results.log_dp_mean
```

Comment:
could you revert the change here. I think we want to remove invalid samples.

Reply:
Samples are already truncated to valid samples in results.